InteractiveHtmlBom/ecad/schema/genericjsonpcbdata_v1.schema

```diff
@@ -0,0 +1,574 @@
+            "enum": [
+                "circle",
+                "oblong"
+            ],
+            "title": "Drillshape"
+        },
+        "Layer": {
+            "type": "string",
+            "enum": [
+                "B",
+                "F"
+            ],
+            "title": "Layer"
+        },
+        "Shape": {
+            "type": "string",
+            "enum": [
+                "rect",
+                "circle",
+                "oval",
+                "roundrect",
+                "chamfrect",
+                "custom"
+            ],
+            "title": "Shape"
+        },
+        "PadType": {
+            "type": "string",
+            "enum": [
+                "smd",
+                "th"
+            ],
+            "title": "PadType"
+        },
+        "Tracks": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "F": { 
+                    "type": "array",
+                    "items": { "$ref": "#/definitions/Track" }
+                },
+                "B": { 
+                    "type": "array",
+                    "items": { "$ref": "#/definitions/Track" }
+                }
+            },
+            "required": [ "F", "B" ],
+            "title": "Tracks"
+        },
+        "Track": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "start": { "$ref": "#/definitions/XY" }, 
+                "end": { "$ref": "#/definitions/XY" }, 
+                "width": { "type": "number" },
+                "net": { "type": "string" }
+            },
+            "required": [
+                "start",
+                "end",
+                "width"
+            ]
+        },
+        "Zones": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "F": { 
+                    "type": "array",
+                    "items": { "$ref": "#/definitions/Zone" }
+                },
+                "B": { 
+                    "type": "array",
+                    "items": { "$ref": "#/definitions/Zone" }
+                }
+            },
+            "required": [ "F", "B" ],
+            "title": "Zones"
+        },
+        "Zone": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "svgpath": { "type": "string" },
+                "polygons": {
+                    "$ref": "#/definitions/Polygons"
+                },
+                "net": { "type": "string" }
+            },
+            "required": [ "svgpath", "polygons" ],
+            "title": "Zone"
+        },
+        "Polygons": {
+            "type": "array",
+            "items": { 
+                "type": "array",
+                "items": { 
+                    "$ref": "#/definitions/XY" 
+                }
+            }
+        },
+        "FontData": {
```

Comment:
I don't think there's any point in having this in schema. As well as allowing text without svgpath.
If there is no svgpath in text objects ibom renders text in kicad specific manner using kicad font which is not useful for other EDAs.